BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -14,6 +14,7 @@ protocol WalletContentView: AnyObject {
 class WalletViewController: UIViewController, RewardsSummaryProtocol {
   
   let state: RewardsState
+  weak var currentNotification: RewardsNotification?
```

Comment:
Why is this `weak`?

Reply:
Since the real owner is ledger, and the ledger may remove it from its stack at any time, I found no real reason to keep it strong. If ledger deletes it then we dont have to `deleteNotification` before loading next